app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,27 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+
+
     <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="My Weather App"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        android:id="@+id/TextView_Title"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/activity_main_text_view_height"
+        android:background="@color/white"
+        android:gravity="center"
+        android:text="@string/app_name"
+        android:textColor="@color/teal_200"
+        android:textStyle="italic"
+        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
this line is not necessary because you are setting the width to a match_parent

Reply:
Done thx!

Reply:
Done thx